File: src/main/java/com/terrencewei/markdown/bean/OSSOutput.java
package com.terrencewei.markdown.bean;

import java.util.Arrays;

/**
 * Created by terrencewei on 2017/04/21.
 */
public class OSSOutput {

    private boolean     success;
    private String      code;
    private String      msg;
    private OSSObject[] objects;



    public boolean isSuccess() {
        return success;
    }



    public void setSuccess(boolean pSuccess) {
        success = pSuccess;
    }



    public String getCode() {
        return code;
    }



    public void setCode(String pCode) {
        code = pCode;
    }



    public String getMsg() {
        return msg;
    }



    public void setMsg(String pMsg) {
        msg = pMsg;
    }



    public OSSObject[] getObjects() {
        return objects;
    }



    public void setObjects(OSSObject[] pObjects) {
        objects = pObjects;
    }



    @Override
    public String toString() {
        return "OSSOutput [" + "success=" + success + ", code=" + code + ", msg=" + msg + ", objects="
                + Arrays.toString(objects) + ']';
    }
}

File: src/main/resources/static/php/my/src/com/terrencewei/markdown/bean/OSSOutput.php
<?php

namespace com\terrencewei\markdown\bean;

class OSSOutput implements \JsonSerializable
{
    private $success;
    private $code;
    private $msg;
    private $objects = array();

    public function jsonSerialize()
    {
        return get_object_vars($this);
    }


    /**
     * @return mixed
     */
    public function getSuccess()
    {
        return $this->success;
    }

    /**
     * @param mixed $success
     */
    public function setSuccess($success)
    {
        $this->success = $success;
    }

    /**
     * @return mixed
     */
    public function getCode()
    {
        return $this->code;
    }

    /**
     * @param mixed $code
     */
    public function setCode($code)
    {
        $this->code = $code;
    }

    /**
     * @return mixed
     */
    public function getMsg()
    {
        return $this->msg;
    }

    /**
     * @param mixed $msg
     */
    public function setMsg($msg)
    {
        $this->msg = $msg;
    }

    /**
     * @return array
     */
    public function getObjects()
    {
        return $this->objects;
    }

    /**
     * @param array $objects
     */
    public function setObjects($objects)
    {
        $this->objects = $objects;
    }


}